e2e-tests/haproxy/run

```diff
@@ -10,7 +10,7 @@ test_default_replicas_service() {
 
 test_disable_replicas_service() {
 	kpatch_set_field pxc $1 "/spec/haproxy/replicasServiceEnabled" false && sleep 1
-	kubectl_bin get svc "$1-haproxy-replicas" 2>&1 | grep -e "not found$" >/dev/null
+	! kubectl_bin get svc "$1-haproxy-replicas"
```

Comment:
I think the previous line was more clear, did you have any problems with it?

Reply:
yes, `errexit` stops script right after `kubectl_bin get svc "$1-haproxy-replicas"` completes

Reply:
```
(kubectl_bin get svc "$1-haproxy-replicas" || :) 2>&1 | grep -e "not found$"
```
what about this command?

Reply:
it works. thanks